### PR TITLE
Mining Vendor Redo

### PR DESCRIPTION
**Pull request by katnep**
basically Re adds all the stuff i added back at the start of 6th port,
also re added stimpacks to the lsit becuase i don't see why they needed
to be removed from it

### CODE REVIEW

File: code/modules/mining/machine_vending.dm [hunk 1]
```diff
@@ -51,6 +51,11 @@
 		new /datum/data/mining_equipment("Drone Ranged Upgrade",/obj/item/device/mine_bot_ugprade/cooldown,								600),
 		new /datum/data/mining_equipment("Drone AI Upgrade",	/obj/item/slimepotion/sentience/mining,									1000),
 		new /datum/data/mining_equipment("Jump Boots",			/obj/item/clothing/shoes/bhop,											2500),
+		new /datum/data/mining_equipment("Survival Knife",		/obj/item/weapon/kitchen/knife/combat/survival,							350),
+		new /datum/data/mining_equipment("Syringe Economy Pack",/obj/item/weapon/storage/box/medipens/survival,							2250),
```
Comment (TalkingCactus) [inline code/modules/mining/machine_vending.dm]: This needs a more descriptive name.

File: code/modules/mining/machine_vending.dm [hunk 2]
```diff
@@ -51,6 +51,11 @@
...
+		new /datum/data/mining_equipment("Stimpack",			/obj/item/weapon/reagent_containers/hypospray/medipen/stimpack,			250),
+		new /datum/data/mining_equipment("Stimpack Economy Pack",/obj/item/weapon/storage/box/medipens/utility,							1250),
+		new /datum/data/mining_equipment("Whetstone",			/obj/item/weapon/sharpener,												400),
```
Comment (TalkingCactus) [inline code/modules/mining/machine_vending.dm]: Whetstones can stay gone.

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 3]
```diff
@@ -179,11 +179,23 @@
 	desc = "A box with several stimpack medipens for the economical miner."
 	illustration = "syringe"
 
+
```
Comment (TalkingCactus) [inline code/game/objects/items/weapons/storage/boxes.dm]: Excessive line spacing, remove one.

**Review by TalkingCactus — CHANGES_REQUESTED** [review]

File: code/modules/mining/machine_vending.dm (hunk 2 above)
Comment (Roland410) [reply code/modules/mining/machine_vending.dm]: Whetstones aren't a REALLY bad thing and I don't understand why they were removed anyways.

File: code/modules/mining/machine_vending.dm (hunk 1 above)
Comment (katnep) [reply code/modules/mining/machine_vending.dm]: done changed to 'survival pen economy pack'

File: code/modules/mining/machine_vending.dm (hunk 2 above)
Comment (katnep) [reply code/modules/mining/machine_vending.dm]: removed

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 3 above)
Comment (katnep) [reply code/game/objects/items/weapons/storage/boxes.dm]: removed the line

File: code/modules/mining/machine_vending.dm [hunk 4]
```diff
@@ -52,10 +52,9 @@
 		new /datum/data/mining_equipment("Drone AI Upgrade",	/obj/item/slimepotion/sentience/mining,									1000),
 		new /datum/data/mining_equipment("Jump Boots",			/obj/item/clothing/shoes/bhop,											2500),
 		new /datum/data/mining_equipment("Survival Knife",		/obj/item/weapon/kitchen/knife/combat/survival,							350),
-		new /datum/data/mining_equipment("Syringe Economy Pack",/obj/item/weapon/storage/box/medipens/survival,							2250),
+		new /datum/data/mining_equipment("Survival Pen Economy Pack",/obj/item/weapon/storage/box/medipens/survival,							2000),
 		new /datum/data/mining_equipment("Stimpack",			/obj/item/weapon/reagent_containers/hypospray/medipen/stimpack,			250),
-		new /datum/data/mining_equipment("Stimpack Economy Pack",/obj/item/weapon/storage/box/medipens/utility,							1250),
-		new /datum/data/mining_equipment("Whetstone",			/obj/item/weapon/sharpener,												400),
+		new /datum/data/mining_equipment("Stimpack Economy Pack",/obj/item/weapon/storage/box/medipens/utility,							1000),
```
Comment (TalkingCactus) [inline code/modules/mining/machine_vending.dm]: Trailing comma, remove it.

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 3 above)
Comment (TalkingCactus) [reply code/game/objects/items/weapons/storage/boxes.dm]: The line is still there.

**Comment by TalkingCactus** [comment]
This will be closed after seven days of inactivity. Please complete or dispute the requested changes.

**Review by TalkingCactus — APPROVED** [review]

**Pull request merged**